pallets/check-membership/src/tight/mod.rs

```diff
@@ -0,0 +1,58 @@
+//! Pallet that demonstrates a minimal access control check. When a user calls this pallet's
+//! only dispatchable function, `check_membership`, the caller is checked against a set of approved
+//! callers. Only if the caller is approved, do they successfully emit the event.
```

Comment:
```suggestion
//! callers. If the caller is a member of the set, the pallet's `IsAMember` event is emitted. Otherwise a `NotAMember` error is returned.
```